src/elements/common/async-load/AsyncLoad.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import * as React from 'react';
+import retry from 'utils/retry';
+
+type Props = {
+    loader: () => Promise<any>,
+    fallback?: React.Element<any>,
+    errorComponent?: React.ComponentType<any>,
+};
+
+type State = {
+    error: ?Error,
+};
+
+const NullComponent = () => null;
```

Comment:
Can we use undefined instead of null? Then you can use lodash/noop

Reply:
its a stateless funcitonal component, so i believe it has to return null

Reply:
undefined is not a valid return value for a react component

Reply:
We should just have this NullComponent as a low level base component inside components.
And then re-used here as well as in FeatureFlag.js